spec/lib/droplet_kit/resources/image_action_resource_spec.rb

```diff
@@ -113,6 +113,11 @@ def json
       expect(action.region).to eq(nil)
       expect(action.region_slug).to eq(nil)
     end
+
+    it_behaves_like 'a paginated index' do
+      let(:fixture_path) {'domain_records/all'}
```

Comment:
Can you switch this to use https://github.com/digitalocean/droplet_kit/blob/master/spec/fixtures/image_actions/all.json please?